### ISSUE TITLE
How to check Python version if more than one python version installed.

### ISSUE THREAD

**Issue by s4-2** [nexB/scancode-toolkit#2435]
### Description

> A brief description of the Documentation Improvement or New Section request.
- In Prerequisites section of Scancode-Toolkit docs there should be some command describing that how one can check the 
  python version if they have more than one python version installed in there local machine.
- Example: _Python 2.7_ 
                  _Python 3.6_
                  _Python 3.9_
 ### Link to Documentation Page

> Where the confusion/inconsistency/incomplete documentation is.
- In Scancode-Toolkit Documents [Prerequisites](https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#before-installing).

### Select Category

-  [ ] Inconsistency
-  [x] New Section Request
-  [x] General Improvement
-  [ ] Typo/Mistakes
-  [ ] Other

**Comment by sritasngh** [comment]
If you want to know the version of python 3 installed on your pc, you can simply write python3 --version command similarly for python2 write python2 --version. Moreover if you want to know your default python version simply write python --version.

**Comment by s4-2** [comment]
Thank you @itssingh , I have python 2 and python 3 in my pc I have tried the command before as you mentioned here but it does not work. That's why I have open this issue. I have also set the env path in environmental variable but than also it is not recognised by the cmd. While writing python in cmd it gives me only one version of python.

**Comment by sritasngh** [comment]
Could you please share the screenshot for the same!

**Comment by s4-2** [comment]
Here is the screenshot, when I tried to run command python2 --version and python3 --version. I have both version in my PC.
![Screenshot (20)](https://user-images.githubusercontent.com/56973448/111461902-ff59cd80-86da-11eb-838e-75bfa142d867.png)
Please check it.

**Comment by Aayush-hub** [comment]
@s4-2 Can you just run command `python` in cmd , and can recheck if python is there in you system path.

**Comment by s4-2** [comment]
@Aayush-hub I have checked it. It show the latest version i.e Python 3.9.
![Screenshot (22)](https://user-images.githubusercontent.com/56973448/111469766-51ebb780-86e4-11eb-8981-bfc2c7ff8daa.png)

**Comment by Aayush-hub** [comment]
@s4-2 Now, you can try running `py -V` or `python -V` in cmd as administrator mode.

**Comment by sritasngh** [comment]
> @s4-2 Now, you can try running `py -V` or `python -V` in cmd as administrator mode.

doesn't it give again the default python version!

**Comment by s4-2** [comment]
@itssingh yes, right.  @Aayush-hub  using python -v it gives the default version which is recently added in the environmental variable

**Comment by Aayush-hub** [comment]
@itssingh @s4-2 Yes it gives default version which is added in env variable. @s4-2 For previous version you can try `python2 -V` in cmd as user and cmd as admin both. you can also check env variable for different python paths. Share screenshots if problem continues.

**Comment by s4-2** [comment]
@Aayush-hub Thank you:) ,  But I have set the path properly then also as I have mentioned before it only gives the latest python version path set in environmental variable

**Comment by pombredanne** [comment]
is this resolved now?

**Comment by s4-2** [comment]
@pombredanne  I have created [PR](https://github.com/nexB/scancode-toolkit/pull/2439) to fix this issue.
But the problem is the 'py' command is not supported by all python version I think.

**Comment by chinyeungli** [comment]
@pombredanne / @AyanSinhaMahapatra 
Do you think this can be closed? We already documented the requirement of python3.9+ in https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#prerequisites , It's likely the user's responsibility to properly configure their Python environment. There are multiple ways to do this—such as using pyenv, virtual environments, or setting it system-wide—and we can't cover every possible setup in different systems.

**Comment by AyanSinhaMahapatra** [comment]
@chinyeungli this documentation on python version is adequate, at most you can add a note at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#prerequisites showing how someone can install a specific version of python using something like: `sudo apt install python3.14 python3.14-venv` and that they can add additional repositories like https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa which have more versions

**Comment by chinyeungli** [comment]
I agree the documentation is adequate, and I don't think adding a note the install specific version of python is need. Closing it.
